### ISSUE TITLE
libre office impress nvda does not read the existence of texts or tables when browsing with tabe  

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#13053]
### Steps to reproduce:
create a simple presentation with text and table
open this presentation in libre office print
walk with tabe 
### Actual behavior:
nvda read
Title 1 Selected Shape Editable 1 of 3 Level 1
Content Placeholder 2 Selected Editable Form 2 of 3 Level 1
edit cursor
Title 1 Selected Shape Editable 1 of 3 Level 1
cursor move
panel
unknown
edit cursor
Drawing View: 2 / 2 Editable Document

Content Placeholder 2 Selected Editable Form 2 of 3 Level 1 
### Expected behavior:
nvda should read
Title 1 selected editable form 1 of 3 level 1 (you should read the text written in this field after this text, in this case the title of the presentation.)
Content Placeholder 2 Selected Editable Form 2 of 3 Level 1
Edit cursor (should now read the text entered in this second field.)

panel
unknown
edit cursor
(it should inform that there is a table in this field, the table could also be read in a running form)
In the powerpoint, when walking with tabe through this presentation, the texts existing in the title and in field 1 are read automatically when gaining focus with the tabe.
In impress the field is only read by pressing enter. 
#### NVDA installed/portable/running from source:
instaled
#### NVDA version:
NVDA version alpha-24184,fd9ae8c4
#### Windows version:
11 22000.318
#### Name and version of other software in use when reproducing the issue:
LibreOffice version 7.3.0.0 impress
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If NVDA add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes
log
[impress.txt](https://github.com/nvaccess/nvda/files/7516552/impress.txt)

**Comment by fernando-jose-silva** [comment]
[1O que é um computador.pptx](https://github.com/nvaccess/nvda/files/7516619/1O.que.e.um.computador.pptx)

**Comment by seanbudd** [comment]
cc @michaelweghorn
